src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -21,65 +21,57 @@
         , and a set of
         <v-tooltip top max-width="20rem" content-class="top-tooltip" transition="fade-transition">
           <template v-slot:activator="{ on }">
-            <span v-on="on" class="tool-tip"> Benefit Company Articles</span>
+            <span v-if="isTypeBcomp" v-on="on" class="tool-tip"> Benefit Company Articles</span>
+            <span v-else-if="isTypeBcCompany" v-on="on" class="tool-tip"><br>BC LTD Articles.</span>
+            <span v-else-if="isTypeBcUlcCompany" v-on="on" class="tool-tip"> Unlimited Liability Company Articles</span>
+            <span v-else-if="isTypeBcCcc" v-on="on" class="tool-tip"> Community Contribution Company Articles</span>
```

Comment:
Either now or in the future, it would be nice to move these tooltips (text), and the articles blurbs below, into the corresponding resource files. This will simplify this component and make it generally more reusable.

(If you choose future, please create a follow-on tech debt ticket. This is not my first choice :) )

Reply:
Sorry about this "late" requirement. We called it out in 14142 but not in 14145.

Reply:
Okay I'll migrate everything into the resource files. No sense creating extra work when I'm here now :)

Reply:
Looks good!